.github/ISSUE_TEMPLATE/release_template.md

```diff
@@ -42,17 +42,22 @@ To aid in understanding the state of the release there is a table with status in
 - [ ] Ensure that all release issues created above are assigned to an owner in the component team.
 - [ ] Increase the build frequency for the this release from once a day (H 1 * * *) to once every hour (H/60 * * * *) in [jenkinsFile](https://github.com/opensearch-project/opensearch-build/blob/main/jenkins/check-for-build.jenkinsfile)
 
-### CI/CD (Feature Freeze) - _Ends __REPLACE_RELEASE-minus-14-days__
+### Campaigns
 
-- [ ] Create Jenkins workflows that run daily snapshot builds for OpenSearch and OpenSearch Dashboards. 
-- [ ] Increment each component version to {{ env.VERSION }} and ensure working CI in component repositories.
-- [ ] Make pull requests to add each component to [manifests/{{ env.VERSION }}/opensearch-{{ env.VERSION }}.yml](/opensearch-project/opensearch-build/tree/main/manifests/{{ env.VERSION }}/opensearch-{{ env.VERSION }}.yml) and [manifests/{{ env.VERSION }}/opensearch-dashboards-{{ env.VERSION }}.yml](/opensearch-project/opensearch-build/tree/main/manifests/{{ env.VERSION }}/opensearch-dashboards-{{ env.VERSION }}.yml) with the corresponding checks.
-- [ ] OpenSearch / OpenSearch-Dashboards core and components teams finalize their features
+__REPLACE with OpenSearch wide initiatives to improve quality and consistency.__
+
+### Release branch readiness - _Ends __REPLACE_RELEASE-minus-14-days__
```

Comment:
Let's Capitalize These Titles Consistently?

I think this should just say "Release Branch"

Reply:
Thanks @dblock. Updated all of following suggestions in latest commit.